src/core/l10n.js

```diff
@@ -123,26 +125,7 @@ l10n["zh-cn"] = l10n.zh;
 
 export const lang = html && html.lang in l10n ? html.lang : "en";
 
-/**
- * @template {Record<string, Record<string, string|Function>>} T
- * @param {T} localizationStrings
- * @returns {T[keyof T]}
- */
-export function getIntlData(localizationStrings) {
-  // Proxy return type is a known bug:
-  // https://github.com/Microsoft/TypeScript/issues/20846
-  // @ts-ignore
-  return new Proxy(localizationStrings, {
-    /** @param {string} key */
-    get(data, key) {
-      const result = data[lang][key] || data.en[key];
-      if (!result) {
-        throw new Error(`No l10n data for key: "${key}"`);
-      }
-      return result;
-    },
-  });
-}
+export { getIntlData };
```

Comment:
Should I avoid the re-export and rename all imports now or in some other PR?
I can do it in https://github.com/w3c/respec/issues/2084

Reply:
I would prefer now 👍